README.md

```
@@ -31,43 +33,60 @@ A Golang client for the configuration center framework [Apollo](https://github.c
 ### Import agollo
 
 ```
-go get -u github.com/apolloconfig/agollo/v5@latest
+go get github.com/apolloconfig/agollo/v6@latest
 ```
 
 ### Initialize agollo
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)